scripts/actions/build-deb/contents/lintian-overrides

```diff
@@ -0,0 +1,14 @@
+# No changelog file at the moment
+yarn-next: debian-changelog-file-missing
+# node-gyp throws this
+yarn-next: python-script-but-no-python-dep
+# Some .js files in node_modules have 0777 permissions instead of 0666
+yarn-next: executable-not-elf-or-script
+# Some node_modules have node in shebang
+yarn-next: unusual-interpreter
+# Third-party licenses
+yarn-next: extra-license-file
+# lib/async.js in node_modules
+yarn-next: embedded-javascript-library
+# No manpage... yet!
+yarn-next: binary-without-manpage
```

Comment:
Copied directly from yarn v1. Need to double check if all of these are still needed - A lot of them were due to `node_modules` junk.

Reply:
Were those tags meant to describe the `yarn` package itself, or what it installed? We didn't have a `node_modules` in the deb file, right? 🤔

Reply:
@arcanis - `lintian` is a linter for Debian packages, and this is a list of the ignored lint rules for the resulting package. These overrides were added before we included a bundle in the Debian package.